### PR TITLE
Feature/country header

### PR DESCRIPTION
**Pull request by danielkopo**

### CODE REVIEW

File: src/Roads/RoadStatusSection.js [hunk 1]
```diff
@@ -2,12 +2,14 @@ import React from 'react';
 import RoadStatus from './RoadStatus';
 import AddTollRoad from './AddTollRoad';
 import './RoadStatusSection.sass';
+import { Card } from 'antd';
 
 const RoadStatusSection = () => (
-    <div className="road-status-section">
+    <Card title="Roads Efficiency" className="road-status-section" bordered={false} style={{ width: 300 }}>
```
Comment (danielkopo) [inline src/Roads/RoadStatusSection.js]: The Card thing is temp until we style this dashboard appropriately.

File: src/App.sass [hunk 2]
```diff
@@ -50,4 +49,4 @@ $third-color: #64dfdf
       border-color: $third-color
 
 .header-logo
-  width: 50px
+  width: 100px
```
Comment (domerd) [inline src/App.sass]: This is way too big

File: src/ConfigCountryModal.js [hunk 3]
```diff
@@ -29,7 +30,7 @@ const ConfigModal = () => {
 
     return (
         <>
-            <Button className="config-modal-button" icon={<SettingOutlined />} onClick={openModal} />
+            <Button className="config-modal-button" shape="circle" icon={<SettingOutlined />} onClick={openModal} />
```
Comment (domerd) [inline src/ConfigCountryModal.js]: Nice

File: src/ConfigCountryModal.sass [hunk 4]
```diff
@@ -0,0 +1,8 @@
+@import './App.sass'
+
+
+.ant-btn
```
Comment (domerd) [inline src/ConfigCountryModal.sass]: This runs on every button, open the modal and check out the buttons there.
You need to add a specific className/ID

File: src/CountryHeader.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    const { getDefaultCountryName } = useContext(CountryStoreContext);
+    const [country, setCountry] = useState(undefined);
+
+    useEffect(() => {
+        setCountry(getDefaultCountryName());
+    }, [getDefaultCountryName]);
+
+    return (
+        <div id="country-container">
+            <Tag style={{ fontSize: 'large', padding: 10 }} color="white">
```
Comment (domerd) [inline src/CountryHeader.js]: Move style to `.sass` file.
Maybe add a prefix: "You are now configuring:" or something like that

File: src/CountryHeader.sass [hunk 6]
```diff
@@ -0,0 +1,8 @@
+@import './App.sass'
+
+#country-container
+  margin: 5px auto auto auto
+
+h1
```
Comment (domerd) [inline src/CountryHeader.sass]: Again, might run on every h1, indent it to run inside the context of `#country-container`

File: src/HomePage.sass [hunk 7]
```diff
@@ -15,16 +15,6 @@
     flex-direction: row
     margin: 30px
 
-  .config-line
```
Comment (domerd) [inline src/HomePage.sass:18]: Left area now looks weird, the button is not aligned and there is no margin between the divider and the statuses

File: src/ResultJson.js [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+import ReactJson from 'react-json-view';
+import { ResultStoreContext } from './ResultStore';
+
+const ResultJson = () => {
+    const { result } = useContext(ResultStoreContext);
+
+    return (
+        <ReactJson
+            style={{
+                borderRadius: 10, padding: 50, textAlign: 'left',
```
Comment (domerd) [inline src/ResultJson.js]: Use the .scss file

**Review by domerd — CHANGES_REQUESTED** [review]
Buttons swithced places (and look weird(:

Screen Shot 2020-07-21 at 10 54 36

File: src/HomePage.sass (hunk 7 above)
Comment (danielkopo) [reply src/HomePage.sass:18]: just removed the divider.
Also canceled the hover resize. I think for now it's not very helpful, even annoying.

**Pull request merged**